src/process.cr

```diff
@@ -129,19 +129,32 @@ class Process
     pid
   end
 
-  # The standard `IO` configuration of a process:
-  #
-  # * `nil`: use a pipe
-  # * `false`: no `IO` (`/dev/null`)
-  # * `true`: inherit from parent
-  # * `IO`: use the given `IO`
-  alias Stdio = Nil | Bool | IO
+  # How to redirect the standard input, output and error IO of a process.
+  enum Redirect
+    # Pipe the IO so the parent process can read (or write) to the process IO
+    # throught `#input`, `#output` or `#error`.
+    PIPE
```

Comment:
I think there are a lot of other places in the stdlib where the enum constants are pascal case, so this should be changed for consistency.